packages/block-editor/src/components/global-styles/use-global-styles-output.js

```diff
@@ -396,9 +396,22 @@ export function getStylesDeclarations(
 			 * Values that already have a "clamp()" function will not pass the test,
 			 * and therefore the original $value will be returned.
 			 */
+			const typographySettings =
+				!! tree?.settings?.typography?.fluid &&
+				tree?.settings?.layout?.wideSize
+					? {
+							fluid: {
+								maxViewPortWidth:
+									tree?.settings?.layout?.wideSize,
+								...tree?.settings?.typography?.fluid,
```

Comment:
Tiniest of nits, and probably no need to worry about it: the ternary for this path means that both `fluid` and `wideSize` are truthy, so I don't think we need the optional chaining on these two lines? (we still do need it below)

Reply:
By gum, you're right! Good spotting. Fixing...